stdlib.sh

```diff
@@ -1068,6 +1068,71 @@ layout_pyenv() {
   [[ -n "$PYENV_VERSION" ]] && export PYENV_VERSION
 }
 
+# Usage: layout uv [<python_version>]
+#
+# Uses uv to create and a virutal environment, and then activates it.
+# See https://docs.astral.sh/uv/pip/environments/ for more details.
+# If no python version is specified, the default python version is used. uv will
+# also install the requested python version if it is not installed.
+layout_uv() {
+    # optional argument to specify python version
+    local python_version=$1
+    if [[ -d ".venv" ]]; then
+        VIRTUAL_ENV="$(pwd)/.venv"
```

Comment:
```suggestion
        VIRTUAL_ENV="${PWD}/.venv"
```
Avoids the sub-shell